os/board/rtl8730e/src/rtl8730e_boot.c

```diff
@@ -239,10 +239,10 @@ void board_gpio_initialize(void)
 		u32 pinmode;
 		u32 pinpull;
 	} pins[] = {
-				{PA_23, PIN_INPUT, PullNone},
+				{PA_23, PIN_INPUT, PullDown},
```

Comment:
1. If rtl8730e_boot.c and ameba_pinmapcfg.c has different setting for the same pin, what happen?
2. This single commit has two changes, pin map configuration change and adding code comment. Due to this, it's difficult to see which pins are changed. Could you split a commit for adding code comment?

Reply:
1.
pinmap_init() is config all pins to the right function and pull state based on pmap_func table.
board_gpio_initialize() is intended to configure specific GPIO pins initialization for board-level functions.

The sequence pinmap_init() -> board_gpio_initialize().
If the pull-up/down configurations set by the two functions are different, usually the latter (board_gpio_initialize()) will override the former, so conflicts should be avoided.
If it is clear that a specific pin is used for GPIO and no other functionality is required, the pull-up/down configuration can be set directly in pinmap_init(), which can reduce unnecessary configurations and conflicts.

2. Done separated to two commits.


Additional Info:
From the email discussion: AIDual and AILite pinmapcfg is not compatible.
Did you have any macro to separate modules pinmap.
Will Samsung take back responsibility for the pinmapcfg management in the future?

Reply:
Hello @ZhenBei-Sin .
For both, I suggest to manage all pin configurations in a single file regardless of pin types (gpio, i2c, i2s and etc).
It's difficult to know where it configures and can be missing.
If not, SS has no plan to do. We can't know where they are for each chipset.